docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -147,61 +155,345 @@ This is the disk volume of a node.
 
-
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-3.png)
 
 ## Step 2. Provision the YugabyteDB nodes
 
 After finishing the cloud provider configuration, click on “Manage Instances” to provision as many nodes as your application requires:
 
-
 ### Manage Instances
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-4.png)
 
-1. Click on “Add Instances” to add the YugabyteDB node. You can use DNS names or IP addresses when adding instances.
-2. Instance ID is an optional user-defined identifier.
+1. Click Add Instances to add the YugabyteDB node. You can use DNS names or IP addresses when adding instances.
+1. Instance ID is an optional user-defined identifier.
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-5.png)
 
-### Run the pre-provisioning script
+### Provision the nodes manually
+
+To provision the nodes manually, you have two options:
+
+* If the SSH user you provided has sudo privileges but requires a password, follow the steps in [Run the pre-provisioning script](#run-the-pre-provisioning-script).
+
+* If the SSH user does not have sudo privileges, follow the steps in [Set up database nodes manually](#set-up-database-nodes-manually).
+
+#### Run the pre-provisioning script
 
 {{< note title="Note" >}}
-This step is only required if you set “Manually Provision Nodes” to true otherwise, you need to skip this step.
+This step is only required if you set “Manually Provision Nodes” to true; otherwise, you need to skip this step.
 {{< /note >}}
 
 Follow these steps for manually provisioning each node by executing the pre-provisioning python script. 
 
-
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-6.png)
 
-1. Login (ssh) to the Platform virtual machine
-2. Access the docker `yugaware `container
+1. Login (ssh) to the Platform virtual machine.
+1. Access the docker `yugaware` container.
+
+    ```sh
+    $ sudo docker exec -it yugaware bash
+    ```
+
+1. Copy/paste the python script prompted in the UI and substitute for a node IP address and mount points. 
+
+    \
+    (Optional) Use the `--ask_password` flag if the sudo user requires password authentication.
+
+    ```output
+    bash-4.4# /opt/yugabyte/yugaware/data/provision/9cf26f3b-4c7c-451a-880d-593f2f76efce/provision_instance.py --ip 10.9.116.65 --mount_points /data --ask_password
+    Executing provision now for instance with IP 10.9.116.65...
+    SUDO password:
+    ```
+
+1. Wait for the script to finish with the SUCCESS status.
+
+1. Repeat step 3 for every node that will participate in the universe.
+
+You’re finished configuring your on-premises cloud provider, and now you can proceed to universe creation. 
+
+#### Set up database nodes manually
+
+{{< note title="Note" >}}
+This step is only required if you set “Manually Provision Nodes” to true; otherwise, you need to skip this step.
```

Comment:
Similar to the previous comment, can we change this to:
This step is only required if you set “Manually Provision Nodes” to true AND the SSH user DOES NOT have sudo privileges; otherwise, you need to skip this step.